noir/tooling/nargo/src/artifacts/contract.rs

```diff
@@ -13,8 +21,26 @@ pub struct ContractArtifact {
     pub functions: Vec<ContractFunctionArtifact>,
     /// All the events defined inside the contract scope.
     pub events: Vec<ContractEvent>,
+    /// Map of file Id to the source code so locations in debug info can be mapped to source code they point to.
+    pub file_map: BTreeMap<FileId, DebugFile>,
+    /// Compilation warnings.
+    pub warnings: Vec<SsaReport>,
```

Comment:
We don't need to store these on the artifact as they're not really relevant after compilation in my mind.